### PR TITLE
Update strainscan to 1.0.13

### PR DESCRIPTION
**Pull request by BiocondaBot**
Update [`strainscan`](https://bioconda.github.io/recipes/strainscan/README.html): **1.0.10** &rarr; **1.0.13**

[install with bioconda](http://bioconda.github.io/recipes/strainscan/README.html) [Conda](https://anaconda.org/bioconda/strainscan/files)

Info | Link or Description
-----|--------------------
Recipe | [`recipes/strainscan`](https://github.com//bioconda/bioconda-recipes/tree/bump/strainscan/recipes/strainscan) (click to view/edit other files)
Summary | One efficient and accurate strain-level microbiome composition analysis tool based on reference genomes and k-mers.
Home | [https://github.com/liaoherui/StrainScan](https://github.com/liaoherui/StrainScan)
Releases |[https://pypi.org/pypi/strainscan/json](https://pypi.org/pypi/strainscan/json)
Recipe Maintainer(s) | @liaoherui

***

This pull request was automatically generated (see [docs](https://bioconda.github.io/contributor/updating.html)).

### CODE REVIEW

**Comment by mencian** [comment]
```
[Sep 28 21:07:04] SERR File "/usr/local/lib/python3.8/site-packages/StrainScan/library/Unique_kmer_detect_direct.py", line 8, in <module>
[Sep 28 21:07:04] SERR import seqpy
[Sep 28 21:07:04] SERR ModuleNotFoundError: No module named 'seqpy'
```

Missing the `seqpy` dependency, which is not in the Bioconda channel yet.

**Comment by liaoherui** [comment]
Thanks for the help! Sorry for this error. I have fixed some bugs in v1.0.12, and will update strainscan to v1.0.13 later. Thus, please ignore this requirement.

**Comment by liaoherui** [comment]
Hi, @mencian , could you please use `- python ==3.7.3` rather than `- python >=3.7` in the "meta.yaml" file? StrainScan can only run under the specific python version. Thanks!

File: recipes/strainscan/meta.yaml [hunk 1]
```diff
@@ -1,28 +1,30 @@
...
+  script: "{{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vvv"
   noarch: python
+  run_exports:
+    - {{ pin_subpackage('strainscan', max_pin="x") }}
 
 requirements:
   host:
     - pip
-    - python ==3.7.3
+    - python >=3.7
```
Comment (mencian) [inline recipes/strainscan/meta.yaml:25]:
```suggestion
    - python ==3.7.3
```

File: recipes/strainscan/meta.yaml [hunk 2]
```diff
@@ -1,28 +1,30 @@
...
   run:
-    - python ==3.7.3
+    - python >=3.7
```
Comment (mencian) [inline recipes/strainscan/meta.yaml:27]:
```suggestion
    - python ==3.7.3
```

**Comment by mencian** [comment]
@liaoherui I have reverted the Python restraints back to ==3.7.3. I referred to the pypi source for your package and was mistaken! My apologies.

**Review by mencian — APPROVED** [review]

**Pull request merged**